data_safe_haven/administration/users/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .user_handler import UserHandler
+from data_safe_haven.administration.users.user_handler import UserHandler
```

Comment:
Why is this type of import better? We're in `__init__.py` here and defining what should be exported from this module. If anything *isn't* in this directory that's a problem, and I feel like this syntax makes it harder to verify that.

Reply:
That is [this rule](https://beta.ruff.rs/docs/rules/relative-imports/). I like it in general. This is a case where it looks odd though.

We can [configure it to allow relative imports of siblings](https://beta.ruff.rs/docs/settings/#flake8-tidy-imports-ban-relative-imports).

Reply:
Happy if you prefer it. I'll let you decide.

Reply:
Let's allow it. I don't see a strong argument to always require absolute imports.

(If we find one, it is easy to change the configuration and let ruff update them all).